unison-cli/src/Unison/Codebase/Editor/HandleInput/MoveAll.hs

```diff
@@ -11,9 +11,9 @@ import Unison.Codebase.Path qualified as Path
 import Unison.HashQualified' qualified as HQ'
 import Unison.Prelude
 
-handleMoveAll :: Bool -> Path.Path' -> Path.Path' -> Text -> Cli ()
-handleMoveAll hasConfirmed src' dest' description = do
```

Comment:
I thought confirmation might not be as necessary with the new reflog tools and since it's not a global root, but I may need to revisit that assumption 🤔